### ISSUE TITLE
Can't run with Gradle 8

### ISSUE THREAD

**Issue by carl-mastrangelo** [reyerizo/jcstress-gradle-plugin#46]
Running with 0.8.14 I get the following exception:

```
* Exception is:
java.lang.NoSuchMethodError: 'org.gradle.api.tasks.JavaExec com.github.erizo.gradle.JcstressTask.setMain(java.lang.String)'
        at com.github.erizo.gradle.JcstressPlugin.addJcstressTask(JcstressPlugin.java:225)
        at com.github.erizo.gradle.JcstressPlugin.apply(JcstressPlugin.java:114)
        at com.github.erizo.gradle.JcstressPlugin.apply(JcstressPlugin.java:64)
        at org.gradle.api.internal.plugins.ImperativeOnlyPluginTarget.applyImperative(ImperativeOnlyPluginTarget.java:43)
        at org.gradle.api.internal.plugins.RuleBasedPluginTarget.applyImperative(RuleBasedPluginTarget.java:51)
        at org.gradle.api.internal.plugins.DefaultPluginManager.addPlugin(DefaultPluginManager.java:187)
        at org.gradle.api.internal.plugins.DefaultPluginManager.access$100(DefaultPluginManager.java:52)
        at org.gradle.api.internal.plugins.DefaultPluginManager$AddPluginBuildOperation.run(DefaultPluginManager.java:282)

```

**Comment by jerzykrlk** [comment]
Good evening Carl. I've released version 0.8.15 - could you confirm that the plugin works well with your setup? Thanks!

**Comment by carl-mastrangelo** [comment]
Thanks for the fast turn around.  Just checked and it works!
